Makefile

```diff
@@ -21,3 +21,10 @@ deps: dev-deps
 dev-deps:
 	go get github.com/golang/lint/golint
 	go get github.com/kisielk/errcheck
+	go get github.com/x-cray/logrus-prefixed-formatter
+	go get github.com/pconcepcion/dice
+	go get gopkg.in/telegram-bot-api.v4
+	go get github.com/x-cray/logrus-prefixed-formatter
```

Comment:
this dependency is duplicated and all this dependencies should go on the deps section as they are dependencies of the program not development dependencies... the problem is that I just copied the Makefile from other project and forgot to update it.